ExportScripts/export_standalone_monolithic_windows.py

```diff
@@ -0,0 +1,294 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of https://www.github.com/o3de/o3de.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+import argparse
+import pathlib
+import logging
+import os
+import time
+import glob
+
+from o3de.validation import valid_o3de_project_json, valid_o3de_engine_json
+from queue import Queue, Empty
+from threading  import Thread
+from typing import List
+from subprocess import Popen, PIPE
+
+logger = logging.getLogger('o3de.gamejam')
```

Comment:
`o3de.gamejam` ? I think something like  `MultiplayerSample Export` might make more sense

Reply:
Missed that one! Thanks for the catch, that'll be updated in a future PR